src/rules/prefer-importing-jest-globals.ts

```diff
@@ -0,0 +1,57 @@
+import globalsJson from '../globals.json';
+import { createRule, parseJestFnCall } from './utils';
+
+export default createRule({
+  name: __filename,
+  meta: {
+    docs: {
+      category: 'Best Practices',
+      description: 'Prefer importing Jest globals',
+      recommended: false,
+    },
+    messages: {
+      preferImportingJestGlobal:
+        "Jest function \"{{ jestFunction }} is used but not imported from '@jest/globals'",
+    },
+    type: 'problem',
+    schema: [],
+  },
+  defaultOptions: [],
+  create(context) {
+    const jestGlobalFunctions = Object.keys(globalsJson);
+    const importedJestFunctions: string[] = [];
+    const usedJestFunctions = new Set<string>();
+
+    return {
+      CallExpression(node) {
+        const jestFnCall = parseJestFnCall(node, context);
+
+        if (!jestFnCall) {
+          return;
+        }
+        if (
+          jestFnCall.head.type === 'import' &&
+          jestGlobalFunctions.includes(jestFnCall.name)
+        ) {
+          importedJestFunctions.push(jestFnCall.name);
+        }
+
+        /* istanbul ignore else */
+        if (jestGlobalFunctions.includes(jestFnCall.name)) {
```

Comment:
You don't need to be doing this as it's handled by `parseJestFnCall`  (that's why the "else" isn't covered as the condition is always met 🙂)

Reply:
Condition removed and Istanbul comment removed 👍